src/vs/workbench/contrib/authentication/browser/actions/manageTrustedExtensionsForAccountAction.ts

```diff
@@ -225,9 +231,16 @@ class ManageTrustedExtensionsForAccountActionImpl {
 		disposableStore.add(quickPick.onDidCustom(() => {
 			quickPick.hide();
 		}));
-		disposableStore.add(quickPick.onDidTriggerItemButton(e =>
-			this._commandService.executeCommand('_manageAccountPreferencesForExtension', e.item.extension.id, providerId)
-		));
+		disposableStore.add(quickPick.onDidTriggerItemButton(e => {
+			const buttonIndex = e.button && e.item.buttons ? e.item.buttons.indexOf(e.button) : -1;
```

Comment:
Instead of doing this check, save the button references as properties on the class and use those in the comparison